src/Catalyst.Core.Modules.Ledger.Tests/IntegrationTests/TruffleTest/package.json

```diff
@@ -20,6 +20,7 @@
   },
   "homepage": "https://github.com/vicnaum/truffleTest#readme",
   "dependencies": {
+    "@catalyst-net-js/truffle-provider": "0.0.9",
```

Comment:
This js library is published under the wrong npm organisation @stxphxn remove this org please and publish under @catalystnetwork